integration/examples/bazel/bazel-genfiles

```diff
@@ -0,0 +1 @@
+/usr/local/google/home/nkubala/.cache/bazel/_bazel_nkubala/0b5f2432d049b24dd37a3f57ec8ef151/execroot/skaffold/bazel-out/k8-fastbuild/genfiles
```

Comment:
integration/examples/bazel/bazel-* should be added to the gitignore and not checked in

Reply:
thanks for catching that!! I'll remove those.

Reply:
https://github.com/GoogleContainerTools/skaffold/pull/1229/files

Reply:
😨 thanks @r2d4